src/FixedDataTableWidthHelper.js

```diff
@@ -13,142 +13,93 @@
 'use strict';
 
 import React from 'React';
+import forEach from 'lodash/forEach';
```

Comment:
Rewritten to use lodash and work over plain objects rather than elements where we peek into the props and then alter props by cloning the element